src/Arcus.Scripting.KeyVault/Scripts/Set-AzKeyVaultSecretFromFile.ps1

```diff
@@ -0,0 +1,23 @@
+# Use this script to upload a certificate as plain text (multiline-support) into Azure Key Vault.
+
+param (
+    [string][Parameter(Mandatory=$true)] $FilePath = $(throw "The path to the file is required."),
+    [string][Parameter(Mandatory=$true)] $KeyVaultName = $(throw "The path to the file is required."),
+    [string][Parameter(Mandatory=$true)] $SecretName = $(throw "The path to the file is required."),
+    [System.Nullable[System.DateTime]][Parameter(Mandatory=$false)] $Expires
+)
+
+Write-Host "Creating KeyVault secret..."
+
+$rawContent = Get-Content $FilePath -Raw
```

Comment:
Maybe best to check if you can find the file and throw an error accordingly

Reply:
Yes, I try to just move it as-is, but yes indeed. This can be improved.

Reply:
Done this.